specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```
@@ -0,0 +1,2448 @@
+          },
+          "500": {
+            "description": "Internal error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Successful Entities request": {
+            "$ref": ".//examples//SuccessfulEntitiesRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/entities/recognition/pii": {
+      "post": {
+        "summary": "Entities containing personal information",
+        "description": "The API returns a list of entities with personal information (\\\"SSN\\\", \\\"Bank Account\\\" etc) in the document. For the list of supported entity types, check <a href=\"https://aka.ms/tanerpii\">Supported Entity Types in Text Analytics API</a>. See the <a href=\"https://aka.ms/talangs\">Supported languages in Text Analytics API</a> for the list of enabled languages.\n",
+        "operationId": "EntitiesRecognitionPii",
+        "consumes": [
+          "application/json",
+          "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ModelVersion"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "name": "domain",
+            "in": "query",
+            "description": "(Optional) if specified, will set the PII domain to include only a subset of the entity categories. Possible values include: 'PHI', 'none'.",
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/StringIndexType"
+          },
+          {
+            "$ref": "#/parameters/PiiCategories"
+          },
+          {
+            "$ref": "#/parameters/MultiLanguageInput"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call results in a list of entities containing personal information returned for each valid document",
+            "schema": {
+              "$ref": "#/definitions/PiiResult"
+            }
+          },
+          "400": {
+            "description": "Bad Request.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "500": {
+            "description": "Internal error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Successful Entity PII request": {
+            "$ref": ".//examples//SuccessfulEntityPIIRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/entities/linking": {
+      "post": {
+        "summary": "Linked entities from a well-known knowledge base",
```

The hyphen between well and known must be removed.

```suggestion
        "summary": "Linked entities from a well known knowledge base",
```